grammar.js

```diff
@@ -775,7 +806,7 @@ module.exports = grammar({
 
     _property_name: $ => choice($.identifier, $.reserved_identifier, $.string, $.number),
 
-    reserved_identifier: $ => choice('get', 'set', 'async')
+    reserved_identifier: $ => choice('get', 'set', 'async', 'abstract', 'module', 'interface'),
```

Comment:
@maxbrunsfeld I was basing this off this list: https://www.w3schools.com/js/js_reserved.asp. I just realized though that `module` is not a JavaScript keyword. I'll move `module` over to the TypeScript grammar, but I think `abstract` and `interface` are good here. What do you think?